README.rst

```diff
@@ -24,6 +24,10 @@ consult https://tulip-control.sourceforge.io/doc/install.html
 The next section describes how to build documentation.
 A test suite is provided under ``tests/``.  Consult the section "Testing" below.
 
+Pip can install the latest development version too::
+
+  pip install https://github.com/tulip-control/tulip-control/archive/master.zip
```

Comment:
Thanks for the observations, I applied the comments. Version confusion is indeed an issue, but the result is identical with using the first command listed in the README (`pip install .`). Both yield a version of 1.4.0. The latter will result in a development version identifier only if `gitpython` is present (in which case the user is aware of development dependencies, or use `gitpython` in their own work).